### PR TITLE
[humanizex] Run humanize tests and fix microseconds bug

### PR DESCRIPTION
**Pull request by daniel-goldstein**
This is two independent commits but I put them in one PR, happy to split if you want.

1. These tests weren't being run in CI I think because of the filename not starting with `test_*`. The microseconds test currently fails on main. I believe this is because the fallback case at the end uses the units of the input and not the smallest possible units (on main you get naturaldelta(0.000001) == 0 seconds not 1 microsecond)

2. Changed second(s) -> s, millisecond(s) -> ms, microsecond(s) -> µs

### CODE REVIEW

File: hail/python/test/hailtop/test_humanizex.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+from hailtop.humanizex import naturaldelta, naturaldelta_msec
```
Comment (daniel-goldstein) [inline hail/python/test/hailtop/test_humanizex.py:1]: hm I swear I `git mv`'d this file…

File: hail/python/hailtop/humanizex.py [hunk 2]
```diff
@@ -13,36 +13,39 @@
...
     (_WEEK, 'week'),
     (_DAY, 'day'),
     (_HOUR, 'hour'),
     (_MINUTE, 'minute'),
-    (_SECOND, 'second'),
-    (_MILLISECOND, 'millisecond'),
-    (_MICROSECOND, 'microsecond'),
+    (_SECOND, 's'),
+    (_MILLISECOND, 'ms'),
+    (_MICROSECOND, 'μs'),
```
Comment (danking) [inline hail/python/hailtop/humanizex.py:32]: this won't display correctly on some browsers unless we set `<meta charset="utf-8">`. It also must appear in the first 1024 bytes. [details](https://www.w3.org/International/questions/qa-html-encoding-declarations). It appears to be set for the website, but we need to set it for web_common. I think that's it though?

**Review by danking — CHANGES_REQUESTED** [review]
x

File: hail/python/test/hailtop/test_humanizex.py (hunk 1 above)
Comment (danking) [reply hail/python/test/hailtop/test_humanizex.py:1]: I'm pretty sure git doesn't store moves, it just deduces them from the diff. If you changed it enough, you fool the logic.

File: hail/python/hailtop/humanizex.py (hunk 2 above)
Comment (daniel-goldstein) [reply hail/python/hailtop/humanizex.py:32]: Interesting, didn't know you could be non-utf-8 these days.

**Review by danking — APPROVED** [review]

**Comment by danking** [comment]
Yeah, when there's no encoding set, Safari defaults to IOS Latin 1

Screenshot 2023-09-28 at 10 14 46

Mu is two bytes in UTF-8, so you get two characters

Screenshot 2023-09-28 at 10 17 09


It happens to me fairly often (like a couple times a year), usually on personal websites. Chrome and Firefox must default to UTF-8 or something because I seem to be in the minority of noticing these things.

**Comment by daniel-goldstein** [comment]
Ya my chromium-based browser shows a µ with no charset. Interesting!

**Pull request merged**